test/leafTest.js

```diff
@@ -0,0 +1,151 @@
+const { BigNumber } = require("ethers")
+const { expect } = require("chai")
+
+describe("Leaf", () => {
+  let leafInstance
+
+  beforeEach(async () => {
+    const LeafStub = await ethers.getContractFactory("LeafStub")
+    leafInstance = await LeafStub.deploy()
+    await leafInstance.deployed()
+  })
+
+  describe("make", async () => {
+    it("constructs a leaf", async () => {
```

Comment:
Worth making the expectation more specific. What do you think about: `constructs a leaf with proper structure (address, creationBlock, id)`?

Reply:
[1f3d993](https://github.com/keep-network/sortition-pools/pull/173/commits/1f3d993631a533dd5a91fdf2bef5bad6dacbf637)